phpunit.xml

```diff
@@ -11,10 +11,9 @@
 	</php>
 	<testsuites>
 		<testsuite>
-			<file>tests/phpunit/tests/two-factor.php</file>
-			<file>tests/phpunit/tests/providers/class.two-factor-dummy.php</file>
-			<!-- This test fails in PHP 5.2. FIDO U2F support is PHP 5.3+ only. -->
-			<file phpVersion="5.3.0" phpVersionOperator=">=">tests/phpunit/tests/providers/class.two-factor-fido-u2f.php</file>
+			<directory suffix=".php">tests/phpunit/tests</directory>
+			<exclude>tests/phpunit/tests/providers/class.two-factor-fido-u2f.php</exclude>
+			<file phpVersion="5.3.0">tests/phpunit/tests/providers/class.two-factor-fido-u2f.php</file>
```

Comment:
Technically not, `phpVersionOperator` defaults to `>=`. You're right though, probably better to be explicit. I'll add it.

I'm having a hard time getting set with an older phpunit (<= 3.6) and PHP 5.2 to figure out why this file is still being included in the older builds. Unless I'm missing something, I can't see why the FIDO tests are still being picked up. (Of course, they clearly are, as the big red X is showing!) :(